pwndbg/commands/heap.py

```diff
@@ -7,60 +7,58 @@
 from typing import List
 from typing import Set
 
-import gdb
 from tabulate import tabulate
 
 import pwndbg
+import pwndbg.aglib.heap
+import pwndbg.aglib.memory
+import pwndbg.aglib.proc
+import pwndbg.aglib.typeinfo
+import pwndbg.aglib.vmmap
 import pwndbg.chain
 import pwndbg.color.context as C
 import pwndbg.color.memory as M
 import pwndbg.commands
-import pwndbg.gdblib.heap
```

Comment:
There are more `pwndbg.gdblib` in this file that can be refactored

Reply:
Yes, and those are coming in the next PR. This PR limits itself to changing only as much as needed in the commands to keep Pwndbg from breaking, because I didn't want the scope of this one to be even larger.